src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -403,11 +404,13 @@ export default {
 							<div class="app-sidebar-header__mainname-container">
 								<!-- main name -->
 								<h2 v-show="!nameEditable"
+									:id="`app-sidebar-vue-${uid}__header`"
+									ref="header"
 									v-linkify="{text: name, linkify: linkifyName}"
 									:aria-label="title"
 									:title="title"
 									class="app-sidebar-header__mainname"
-									:tabindex="nameEditable ? 0 : undefined"
+									:tabindex="nameEditable ? 0 : -1"
```

Comment:
```suggestion
									:tabindex="nameEditable ? 0 : undefined"
```
Setting tabindex="-1" will make it non-sequentially focusable but still click focusable which I don't think is desired

Reply:
It is actually important to have `-1` here. This element should not be in the keyboard focus sequence because it is not an interactive element.

But having no `tabindex` is also not correct, because this element receives programmatic focus.

`-1` allows making an element programmatically focusable without adding it to the Tab sequence.

For example, without it, there will be no outline on focus on `<h2>` and it will be broken in the `focus-trap`.

See also:

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/83ed3a8c-6b59-4bc0-bc9e-3d889ce7413c)

Reply:
Makes sense 👍

Reply:
I actually spend hours trying to understand what can I focus here, and why the header didn't work =D

Reply:
That's how the tabindex crumbles 😢